block/manager.go

```diff
@@ -339,11 +352,11 @@ func (m *Manager) trySyncNextBlock(ctx context.Context, daHeight uint64) error {
 	var commit *types.Commit
 	currentHeight := m.store.Height() // TODO(tzdybal): maybe store a copy in memory
 
-	b1, ok1 := m.syncCache[currentHeight+1]
+	b1, ok1 := m.fcr.Apply(m.syncCache[currentHeight+1])
```

Comment:
sync to the latest main, as this logic will change a bit: https://github.com/rollkit/rollkit/blob/main/block/manager.go#L313